skfda/exploratory/visualization/_ddplot.py

```diff
@@ -46,7 +56,11 @@ def __init__(
         dist1: T,
         dist2: T,
         depth_method: Depth[T],
+        chart: Union[Figure, Axes, None] = None,
+        fig: Optional[Figure] = None,
+        axes: Optional[Axes] = None,
```

Comment:
I still think that the chart parameter should be removed... but if it is not it should be a positional parameter (and most parameters should NOT be positional).


```suggestion
        dist1: T,
        dist2: T,
        chart: Union[Figure, Axes, None] = None,
        *,
        depth_method: Depth[T],
        fig: Optional[Figure] = None,
        axes: Optional[Axes] = None,
```

Reply:
ok, i will do this in every visualization method